### DISCUSSION TITLE
Tooltip inside popover opens when popover opens

### DISCUSSION THREAD

**Discussion by a0ax** [huntabyte/shadcn-svelte#923] [Help] [answered]
I have a tooltip component inside a select built with a popover and command menu. When the popover opens, the first tooltip opens. I don't want this to happen. What could I do to prevent the tooltip from opening?

**Comment by vitormisumi** [comment]
I have the same problem with a tooltip inside a dialog

**Comment by justindomingue** [comment]
Possibly because the dialog finds the first focusable element on open

**Reply by a0ax** [reply]
Right, wondering how to prevent this functionality.

**Comment by vitormisumi** [comment, marked as answer]
Passing ignoreNonKeyboardFocus to Tooltip.Root seems to fix it

**Comment by a0ax** [comment]
great beautiful
